avocado/plugins/yaml_to_mux.py

```diff
@@ -413,9 +415,9 @@ def str_variants(self):
         out += tree_repr
         out += "\n\n"
 
-        for variant_id, tpl in self:
-            paths = ', '.join([x.path for x in tpl[0]])
-            out += 'Variant %s:    %s\n' % (variant_id, paths)
+        for variant in self:
+            paths = ', '.join([x.path for x in variant["variant"]])
+            out += 'Variant %s:    %s\n' % (variant["variant_id"], paths)
```

Comment:
With this commit, running `avocado multiplex -m <yaml_file>` results in:

```
Avocado crashed:
Traceback (most recent call last):
  File "./scripts/avocado", line 83, in <module>
    sys.exit(app.run())
  File "/home/cleber/src/avocado/avocado/avocado/core/app.py", line 82, in run
    return method(self.parser.args)
  File "/home/cleber/src/avocado/avocado/avocado/plugins/multiplex.py", line 86, in run
    for (index, tpl) in enumerate(mux.variants):
AttributeError: 'Mux' object has no attribute 'variants'
```

Which is clearly related to this change of variant format.  But since the final version works, it looks like you fixed it in more recent commits.